packages/block/test/eip4844block.spec.ts

```diff
@@ -17,249 +17,253 @@ import { Block } from '../src/index.js'
 import gethGenesis from './testdata/4844-hardfork.json'
 
 import type { TypedTransaction } from '@ethereumjs/tx'
-// Hack to detect if running in browser or not
-const isBrowser = new Function('try {return this===window;}catch(e){ return false;}')
 
-if (isBrowser() === false) {
-  try {
-    initKZG(kzg, __dirname + '/../../client/src/trustedSetups/official.txt')
-  } catch {
-    // no-op
-  }
-}
-
-const common = Common.fromGethGenesis(gethGenesis, {
-  chain: 'customChain',
-  hardfork: Hardfork.Cancun,
-  customCrypto: { kzg },
-})
+describe('EIP4844 header tests', () => {
```

Comment:
i assume all the diff here is because of formatting after removal of isBrowser :+1:

Reply:
Correct